library/src/main/java/org/xmtp/android/library/messages/PrivateKey.kt

```diff
@@ -77,7 +78,7 @@ class PrivateKeyBuilder : SigningKey {
     override val address: String
         get() = privateKey.walletAddress
 
-    override fun sign(data: ByteArray): SignatureOuterClass.Signature {
+    override suspend fun sign(data: ByteArray): SignatureOuterClass.Signature {
```

Comment:
@nplasterer I think we'll have to make these suspend functions since it takes some async work to create the signature from the wallet connect kit unless you have a better idea!